internal/storage/bucket_handle.go

```diff
@@ -179,3 +180,37 @@ func (b *bucketHandle) CopyObject(ctx context.Context, req *gcs.CopyObjectReques
 	o = storageutil.ObjectAttrsToBucketObject(objAttrs)
 	return
 }
+
+func (b *bucketHandle) ListObjects(ctx context.Context, req *gcs.ListObjectsRequest) (listing *gcs.Listing, err error) {
+	// Converting *ListObjectsRequest to type *storage.Query as expected by the Go Storage Client.
+	query := &storage.Query{
+		Delimiter:                req.Delimiter,
+		Prefix:                   req.Prefix,
+		Projection:               storage.Projection(req.ProjectionVal),
```

Comment:
why is projectionVal conversion not handled here
https://github.com/GoogleCloudPlatform/gcsfuse/blob/5651aa43ab0f75f56bb5c3353df60d9c950c75a9/vendor/github.com/jacobsa/gcloud/gcs/bucket.go#L285

Reply:
fixed